rotkehlchen/tests/data/blockfi-trades.csv

```diff
@@ -0,0 +1,2 @@
+Trade ID,Date,Buy Quantity,Buy Currency,Sold Quantity,Sold Currency,Rate Amount,Rate Currency,Type
```

Comment:
I assume all data are taken from somewhere proper?

Reply:
I got it after asking the user for confirmation in some questions #1674